package.json

```diff
@@ -32,9 +35,10 @@
     "install": "^0.12.2",
     "mocha": "^6.2.0",
     "npm": "^6.9.0",
+    "run-script-os": "^1.0.7",
     "sinon": "^7.4.1",
     "sinon-chai": "^3.3.0",
-    "tabris": "^3.3.0-dev",
+    "tabris": "nightly",
```

Comment:
Intended? Not sure if it's a good idea to use "nightly", since the build will break when a new incompatible nightly is released?

Reply:
In that case it SHOULD brake.